libnavigation-core/src/main/java/com/mapbox/navigation/core/fasterroute/FasterRouteDetector.kt

```diff
@@ -5,7 +5,16 @@ import com.mapbox.navigation.base.trip.model.RouteProgress
 
 internal object FasterRouteDetector {
 
+    /**
+     * This threshold is to filters slightly faster routes. For example, if you're 40 minutes
+     * away and there is an alternative that is 1 minute faster - we will ignore it.
+     */
+    private const val PERCENTAGE_THRESHOLD = 0.90
```

Comment:
Is there a way to associate these types of parameters with an access token? Customers would love control of this from a server.

Reply:
@andrewychen

Reply:
maybe it's better to expose this interface to customers. let them create their own implementation of faster route if they want.

Reply:
ok. `NavigationOptions` seems like it'll become really large if every configuration idea like this is added 🤔 

interesting we allowed the override in legacy! I wonder if anyone tried to do that?

Reply:
> ok. `NavigationOptions` seems like it'll become really large if every configuration idea like this is added 🤔 

Yeah, that's a known "issue". Any ideas on how to solve it?

> interesting we allowed the override in legacy! I wonder if anyone tried to do that?

Yeah, haven't heard from anyone so who knows... 😅

Reply:
> ok. NavigationOptions seems like it'll become really large if every configuration idea like this is added 🤔
> Yeah, that's a known "issue". Any ideas on how to solve it?

Making a per feature options is an idea. so for this feature, it'd be `FasterRouteOptions`